SimPEG/PF/Gravity.py

```diff
@@ -19,115 +19,66 @@ class GravityIntegral(Problem.LinearProblem):
     # surveyPair = Survey.LinearSurvey
     forwardOnly = False  # Is TRUE, forward matrix not stored to memory
     actInd = None  #: Active cell indices provided
-    rtype = 'z'
+    rxType = 'z'
```

Comment:
would you mind adding docstrings for these? also, as much as reasonable, lets try and stick with more pep8 names
`rx_type`, `progress_index`